src/Search/Search.tsx

```diff
@@ -28,13 +29,17 @@ const Search: React.FC = () => {
 	}
 
   return (
-		<input 
-			aria-label="search-input"
-			className="search-input"
-			value={query}
-			onChange={(e) => setQuery(e.target.value)}
-			placeholder="Search city, zip, or county"
-		/>
+		<div className="search-bar">
+			<input 
+				aria-label="search-input"
+				className="search-input"
+				value={query}
+				onChange={(e) => setQuery(e.target.value)}
+				placeholder="Search city, zip, or county"
+			/>
+			<img src={magGlass} alt='magnifying glass search icon' className='search-icon'/>
```

Comment:
Can we make this a real button? I found it will be so helpful to fire a function when a button click than the timer on the keystrokes. Then, it can pass "actions" up from Search to App and "data" back down to Location!!!